ibmvmc/ibmvmc.c

```diff
@@ -420,6 +528,24 @@ static int ibmvmc_return_hmc(struct ibmvmc_hmc *hmc, bool bReleaseReaders)
 	return 0;
 }
 
+/**
+ * ibmvmc_send_open - Interface Open
+ * @buffer: Pointer to ibmvmc_buffer struct
+ * @hmc: Pointer to ibmvmc_hmc struct
+ *
+ * This command is sent by the management partition as the result of a
+ * management partition device request. It causes the hypervisor to
+ * prepare a set of data buffers for the LPM connection indicated HMC idx.
```

Comment:
I would recommend search/replace LPM with HMC.  LPM was older terminology that was later changed, but the document was never updated.  I think HMC would be less confusing.